util/src/Pos/Util/Util.hs

```diff
@@ -434,6 +437,23 @@ median l = NE.sort l NE.!! middle
 sleep :: MonadIO m => NominalDiffTime -> m ()
 sleep n = liftIO (threadDelay (truncate (n * 10^(6::Int))))
 
+-- | temporary reimplementation of 'ords' from "Formatting"
+--   because the original function converts the integer value to a real number
+intords :: (Show n, Integral n) => F.Format r (n -> r)
+intords = F.later go
```

Comment:
Has this been submitted upstream? Even as a but report?

Reply:
yes, by Michael Bishop.
we have implemented this locally in 'core', but there were more instances of wrong formatting in other places. This is why it is moved to 'util'.
The problem is that grepping the logs for something like '4th epoch' will not hit the (wrongly formatted) entry '4.0th epoch'.